### DISCUSSION TITLE
Astroid Framework Version 3.1.1

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#735] [Announcements]
We are excited to announce the release of Astroid 3.1.1.

- For more information please visit [astroidframe.work](https://astroidframe.work/). 
- To view the preview page please visit [demo.astroidframe.work](https://demo.astroidframe.work/)

### Added

- [Add Video Widget](https://github.com/templaza/astroid-framework/commit/b0df665f3f69c662cceca64f7d2d03e81a8d6a20)

### Updated

- [Update default layout and Media content widget](https://github.com/templaza/astroid-framework/commit/39757b01df7699c5a71dbe984a686b8370b5c433)
- [Update itemscope for article](https://github.com/templaza/astroid-framework/commit/894956e026dc35f81450963cb0666419111e9af1)
- [Update List widget language](https://github.com/templaza/astroid-framework/commit/1acec6e421b14669012bda5b760e2d684035165f)
- [Update field-name description](https://github.com/templaza/astroid-framework/commit/b862971fbaf32e446132f75d51ba35f63afd1a6f)
- [Add verticle align middle to Grid Widget](https://github.com/templaza/astroid-framework/commit/d4135bc33a33f674706f76b9bd5a2b9bea710139)

### Improved

- [Improve HTML Structure](https://github.com/templaza/astroid-framework/commit/2df408cd731bb3157d66976ecf4538e751e62d4d)
- [Improve Testimonial Widget style.](https://github.com/templaza/astroid-framework/commit/028f05a0a71b796763351a0ecf844f8584a9631e)
- [Replace $_FILE by Joomla input->file->get function](https://github.com/templaza/astroid-framework/commit/0a9d6759f627f61d458b28e3c52dbd2e5bf34e3f)

### Fixed Bugs

- [Fix issue Google Font Not removed from code, when deactivated](https://github.com/templaza/astroid-framework/commit/3d897772e674da967bf2fe5676a68e7f2d84e413) https://github.com/templaza/astroid-framework/issues/730
- [Fix issue wrong label on List Widget](https://github.com/templaza/astroid-framework/commit/1f91ce18f95edc8d9fa757697e99672577c3fdc0)
- [Fix issue warning about $icon_size at Testimonial](https://github.com/templaza/astroid-framework/commit/744efb4da5d03ceac5dc2e2b64ac73e21033e57c)
- [Fix issue load category param when it is null](https://github.com/templaza/astroid-framework/commit/5e6a280dd2d14b51ef8172cfb5de8554a458bbfd)

<hr /><em>This discussion was created from the release <a href='https://github.com/templaza/astroid-framework/releases/tag/v3.1.1'>Astroid Framework Version 3.1.1</a>.</em>

**Comment by Banuk1616** [comment]
Many thanks for all the work and the new Astroid version.

**Comment by paurray** [comment]
So much awesome stuff going on with Astroid it is hard top keep up :-)
Tested briefly the new Video Widget implementation. Wow and thank you.
First impressions seems as always rock solid.
